src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -47,14 +47,25 @@
 public class SurfaceNamer extends NameFormatterDelegator {
   private ModelTypeFormatter modelTypeFormatter;
   private TypeNameConverter typeNameConverter;
+  protected String packageName;
```

Comment:
addendum - as per my comment in RubySurfaceNamer, this should go away completely.